server/svix-server/src/core/types.rs

```diff
@@ -427,6 +428,20 @@ impl Validate for EventTypeNameSet {
     }
 }
 
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+pub struct RetrySchedule(pub Vec<u64>);
```

Comment:
```suggestion
pub struct RetrySchedule(pub Vec<Duration>);
```
Will simplify everything.

Reply:
Unfortunately not -- this approach is simpler than serializing/deserializing Durations.